components/modelmeshserving/modelmeshserving.go

```diff
@@ -19,7 +19,8 @@ import (
 var (
 	ComponentName = "model-mesh"
 	Path          = deploy.DefaultManifestPath + "/" + ComponentName + "/overlays/odh"
-	// monitoringPath         = deploy.DefaultManifestPath + "/" + "modelmesh-monitoring/base"
+	// monitoringPath         = deploy.DefaultManifestPath + "/" + "modelmesh-monitoring/base".
```

Comment:
This is the error for the comment
```components/modelmeshserving/modelmeshserving.go:22:2: comment on exported var `DependentComponentName` should be of the form `DependentComponentName ...` (golint)```

Reply:
and should we consider removing the comment?

Reply:
I would, but lets wait for the feedback from whoever did that in the first place :)

Reply:
@VaishnaviHire  @zdtsw  -> do we need this comment here

Reply:
i explicitly kept this commented-out code for the current release.(same as the end of the file)
the reason is because in ODH the function is not needed as dead-code, but in downstream we will not remove it till next release, To not confuse when doing sync from ODH to downstream, i prefer keeping it like this way till we have downstream ships out then we do the cleanup.

Reply:
once https://github.com/opendatahub-io/opendatahub-operator/pull/703 gets merged, this should not be a problem.

Reply:
let me do a rebase and then this issue is done 👍